examples/kubernetes/connectivity-check/connectivity-check-ipv6.yaml

```diff
@@ -0,0 +1,1338 @@
+# Automatically generated by Makefile. DO NOT EDIT
```

Comment:
This doesn't seem to be automatically generated but a copy of the generated yaml that is then modified. This means that this will get oudated quickly as it will not receive the same fixes. Can we instead automatically generate this from the IPv4 version?

Reply:
agreed, I wait for #12599 then will rebase and add script to generate this.

Reply:
added cue script to generate this file :)